nailgun/entities.py

```diff
@@ -4286,6 +4303,11 @@ def path(self, which=None):
                 super(Host, self).path(which='base'),
                 which
             )
+        elif which in ('bulk/remove_subscriptions',):
+            return '{0}/{1}'.format(
+                super(Host, self).path(which='base'),
+                which
+            )
```

Comment:
I feel this entire section from line 4296 should be in single branch:
```python
elif which in ('bulk/install_content', 'bulk/add_subscriptions', 'bulk/remove_subscriptions'):
            return '{0}/{1}'.format(
                super(Host, self).path(which='base'),
                which
            )
```

Reply:
OK, looks neat

changes made
thank you